EDDiscovery/3DMap/MapManager.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Windows.Forms;
+using EDDiscovery;
+
+
+namespace EDDiscovery2._3DMap
+{
+    public class MapManager
```

Comment:
This is the map proxy I spoke of. It's mostly just there to manage the `show` through the MapForm instance.